honeycomb-beeline.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "appraisal"
   spec.add_development_dependency "bump"
   spec.add_development_dependency "bundler"
+  spec.add_development_dependency "opentelemetry-api"
```

Comment:
this would have to use the `add_dependency` not `add_development_dependency` as it looks like it is used in the actual gem not just tests?

Reply:
We would also need to specify a version that we are compatible with. Does it have a stable API yet?

Reply:
good point!